packages/typespec-python/test/azure/mock_api_tests/asynctests/test_azure_arm_models_resource_async.py

```diff
@@ -209,3 +209,76 @@ async def test_nested_proxy_resources_list_by_top_level_tracked_resource(client)
         assert result.name == "nested"
         assert result.type == "Azure.ResourceManager.Models.Resources/topLevelTrackedResources/top/nestedProxyResources"
         assert result.system_data.created_by == "AzureSDK"
+
+
+@pytest.mark.asyncio
```

Comment:
@ChenxiJiang333 Please run `pnpm change add` to add changelog.

Reply:
@msyyc updated